tests/tests/swfs/avm2/pixelbender_effect_glassDisplace_shaderfilter/test.toml

```diff
@@ -2,6 +2,7 @@ num_frames = 10
 
 [image_comparison]
 tolerance = 1
+max_outliers = 484
```

Comment:
Same here

Reply:
Same result as in the other test, unfortunately